src/analyze/analyze.c

```diff
@@ -229,6 +230,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  security [UNIT...]         Analyze security of unit\n"
                "  inspect-elf FILE...        Parse and print ELF package metadata\n"
                "  malloc [D-BUS SERVICE...]  Dump malloc stats of a D-Bus service\n"
+               "  fdstore [SERVICE...]       Show file descriptor store contents of service\n"
```

Comment:
at least one service is mandatory, so `fdstore SERVICE...` no?

Reply:
true